test/src/sdk-harness/test_projects/ec_recover_and_match_predicate/mod.rs

```diff
@@ -95,9 +95,9 @@ async fn ec_recover_and_match_predicate_test() -> Result<()> {
         .await?;
 
     let predicate_balance = provider
-        .get_asset_balance(predicate.address(), asset_id)
+        .get_asset_balance(&predicate.address(), &asset_id)
         .await?;
-    assert_eq!(predicate_balance, amount_to_predicate);
+    assert_eq!(predicate_balance, amount_to_predicate as u128);
```

Comment:
nit: maybe just set `amount_to_predicate` to `let amount_to_predicate = 1000u128`

Reply:
Then `wallet_1.transfer(..)` would need the conversion.

Reply:
ah yeap; maybe the sdk needs to be updated accordingly; probably future release